seed/lib/mappings/test_mapping_data.py

```diff
@@ -29,11 +29,11 @@ def test_mapping_data_init(self):
 
         # verify that the data loaded as expected
         fake_data_0 = {
-            "js_type": "",
+            "js_type": u"",
             "name": "address_line_1",
             "schema": "BEDES",
             "table": "PropertyState",
-            "type": "CharField",
+            "type": u"CharField",
```

Comment:
no we don't need to coerce it. We flip/flop on this syntax and I don't think it really matters at the moment.